diracx/values.yaml

```diff
@@ -183,12 +183,12 @@ diracxWeb:
   repoURL: ""
   branch: ""
   # -- resources to use for building the webapp if diracxWeb.repoURL is set
-  # installDiracxWeb:
-  #   resources:
-  #    requests:
-  #      memory: 512Mi
-  #    limits:
-  #      memory: 1Gi
+  installDiracxWeb:
```

Comment:
I would just remove the `installDiracxWeb` and put `resources` directly under `diracxWeb`. Otherwise it's fine with me.